controllers/ingressnodefirewall_controller.go

```diff
@@ -324,17 +323,6 @@ func (r *IngressNodeFirewallReconciler) buildNodeStates(
 				continue withNextNode
 			}
 			for _, iface := range firewallObj.Spec.Interfaces {
-				// Verify if an invalid interface name was specified.
-				// On error, report the error in the status field and continue with the next node.
-				if !isLegalInterfaceName(iface) {
```

Comment:
I have a feeling that this will break one of the unit tests but if it does I'll fix it in my PR

Reply:
no unit-test is passing fine I unless it flakes ?